bin/jenkins/functions.sh

```diff
@@ -124,33 +124,45 @@ selenium_tests () {
   return $rc
 }
 
+deploy () {
+  PROJECT=$1
+  print_line
+  echo "Running Deploy"
+  docker exec -i ${PROJECT}_dev_1 su vagrant -c "
+    source /vagrant/bin/init_vagrant_env &&
+    make clean_appengine > /dev/null &&
```

Comment:
Are you sure about all these `/dev/null`? Shouldn't we output to some files in case build fails?